### PR TITLE
Add next to the bump script

### PR DESCRIPTION
**Pull request by jtpio**
## References

This option was missing from the bump script.

Follow-up to #960.

## Code changes

Update to the version bump script.

## User-facing changes

None

## Backwards-incompatible changes

None

### CODE REVIEW

**Comment by trungleduc** [comment]
just curious, why does it still work in the previous PR with `next` spec?

File: scripts/bump-version.py [hunk 1]
```diff
@@ -94,6 +90,10 @@ def bump(force, spec):
     if len(status) > 0:
         raise Exception("Must be in a clean git state with no untracked files")
 
+    # Make sure we have a valid version spec.
+    if spec not in OPTIONS:
+        raise Exception(f"Version spec must be one of: {OPTIONS}")
```
Comment (trungleduc) [inline scripts/bump-version.py:95]:
```suggestion
        raise ValueError(f"Version spec must be one of: {OPTIONS}")
```

**Comment by trungleduc** [comment]
LGTM, thanks!

**Comment by jtpio** [comment]
@meeseeksdev please backport to 0.2.x

**Pull request merged**